README.md

```diff
@@ -17,6 +17,9 @@ Building HDFS-Mesos
 2. Run `./bin/build-hdfs nocompile` to skip the `gradlew clean package` step and just re-bundle the binaries.
 3. To remove the project build output and downloaded binaries, run `./bin/build-hdfs clean`
 
+**NOTE:** The build process builds the artifacts under the `$PROJ_DIR/build` directory.  A number of zip and tar files will be there which are cached for faster subsequent builds.   The tarball used for installation is hdfs-mesos-x.x.x.tgz which contains the scheduler and the executor to be distributed.
+
+
 Installing HDFS-Mesos on your Cluster
 --------------------------
```

Comment:
the details for uploading for install are in the next section of the readme.  I've updated that to be clear and to make it consistent with the new build process.